README.md

```diff
@@ -29,9 +28,9 @@ From simple pilots to production-ready web applications in no time. No more comp
     <br />
     <a href="https://docs.taipy.io/en/latest/"><strong>📚 Explore the docs </strong></a>
     <br />
-    <a href="https://discord.com/invite/SJyz2VJGxV">  🫱🏼‍🫲🏼 Discord support</a>
+    <a href="https://discord.com/invite/SJyz2VJGxV"><strong>  🫱🏼‍🫲🏼 Discord support </strong></a>
```

Comment:
Why isn't this a regular Markdown link like everywhere else?
`[** 🫱🏼‍🫲🏼 Discord support **](https://discord.com/invite/SJyz2VJGxV)`

Same comment on the other instances.